handlers/features.go

```diff
@@ -697,3 +698,45 @@ func (oh *featureHandler) BriefSend(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(resp.StatusCode)
 	w.Write(respBody)
 }
+
+func (oh *featureHandler) GetTickets(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	pubKeyFromAuth, _ := ctx.Value(auth.ContextKey).(string)
+	if pubKeyFromAuth == "" {
+		fmt.Println("no pubkey from auth")
+		w.WriteHeader(http.StatusUnauthorized)
+		return
+	}
+
+	featureUuid := chi.URLParam(r, "feature_uuid")
+	phaseUuid := chi.URLParam(r, "phase_uuid")
+
+	if _, err := uuid.Parse(featureUuid); err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(map[string]string{"error": "Invalid feature UUID format"})
+		return
+	}
+
+	if _, err := uuid.Parse(phaseUuid); err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(map[string]string{"error": "Invalid phase UUID format"})
+		return
+	}
+
+	_, err := oh.db.GetFeaturePhaseByUuid(featureUuid, phaseUuid)
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+		json.NewEncoder(w).Encode(map[string]string{"error": "Phase not found"})
+		return
+	}
+
+	tickets, err := oh.db.GetTicketsByPhase(featureUuid, phaseUuid)
```

Comment:
Also based on the issue related to this, you are suppose to return only the `ticketUuids` and not the entire ticket object

Reply:
Hmm this should have been updated and the approach is corrrect. 

Initially I had a design to return uuids and then use a different operation to get all details. 
But it makes more sense to just return the whole object in one go whilst we're there.

Reply:
This was a situation where the description in the ticket was correct but I'd not updated the title which still referenced UUIDs (updated)